pkg/loghttp/push/otlp.go

```diff
@@ -37,20 +37,21 @@ const (
 
 func newPushStats() *Stats {
```

Comment:
nit: I know this one was already here, but I think it makes more sense to move this new function to push.go right below the definition of Stats

Reply:
lol good find, totally overlooked that stats were defined in the other file. fixed.